src/ValueFormatters/QuantityFormatter.php

```diff
@@ -26,6 +27,24 @@ class QuantityFormatter extends ValueFormatterBase {
 	const OPT_SHOW_UNCERTAINTY_MARGIN = 'showQuantityUncertaintyMargin';
 
 	/**
+	 * Value for the OPT_SHOW_UNCERTAINTY_MARGIN indicating that the uncertainty margin
+	 * should not be shown.
+	 */
+	const SHOW_UNCERTAINTY_MARGIN_NEVER = 'never';
+
+	/**
+	 * Value for the OPT_SHOW_UNCERTAINTY_MARGIN indicating that the uncertainty margin
+	 * should be shown if we are displaying a QuantityValue.
+	 */
+	const SHOW_UNCERTAINTY_MARGIN_IF_KNOWN = 'if-known';
+
+	/**
+	 * Value for the OPT_SHOW_UNCERTAINTY_MARGIN indicating that the uncertainty margin
+	 * should be shown if we are displaying a non-exact QuantityValue.
+	 */
+	const SHOW_UNCERTAINTY_MARGIN_IF_NOT_ZERO = 'if-not-zero';
```

Comment:
This means we will mix camel case with dash style: `showQuantityUncertaintyMargin=if-not-zero`. I wonder if we should have a style guide for this.